lib/book.py

```diff
@@ -83,145 +88,201 @@ def __init__(self, db, book_id, goodreads_conn, amazon_conn, formats, send_to_de
         self._create_send_end_actions = create_send_end_actions
         self._file_preference = file_preference
 
-        self._status = self.IN_PROGRESS
+        self._status = IN_PROGRESS
         self._status_message = None
 
-        self._book_settings = BookSettings(self._db, self._book_id, self._goodreads_conn, amazon_conn, expand_aliases)
+        self._aliases = None
+        self._device_sdr = None
+        self._local_book_directory = None
+        self._formats_on_device = None
+        self._xray_format_information = None
+        self._xray_status = None
+        self._xray_status_message = None
+        self._xray_send_fmt = None
+        self._xray_send_status = None
+        self._xray_send_status_message = None
+        self._author_profile_status = None
+        self._author_profile_status_message = None
+        self._author_profile_send_status = None
+        self._author_profile_send_status_message = None
+        self._start_actions_status = None
+        self._start_actions_status_message = None
+        self._start_actions_send_status = None
+        self._start_actions_send_status_message = None
+        self._end_actions_status = None
+        self._end_actions_status_message = None
+        self._end_actions_send_status = None
+        self._end_actions_send_status_message = None
+        self._goodreads_author_profile = None
+        self._goodreads_end_actions = None
+        self._goodreads_start_actions = None
+        self._goodreads_xray = None
+
+        self._book_settings = BookSettings(self._database, self._book_id, self._goodreads_conn, amazon_conn, expand_aliases)
 
         self._get_basic_information()
 
-        if self._status != self.FAIL:
-            self._status = self.SUCCESS
+        if self._status != FAIL:
+            self._status = SUCCESS
 
     @property
     def status(self):
+        '''Returns _status object.'''
```

Comment:
You've done it now, so may be you want to keep it, but in my experience this sort of comment is both worthless (it is obvious to anyone that knows python what it does) and will be C&P elsewhere without being updated and so will just make the code messy for no benefit.  Is there a way to selectively disable the requirement for docstrings?  Perhaps disabling them for properties globally?